e2e/app/component/concept-domain-card.ts

```diff
@@ -40,7 +39,7 @@ export default class ConceptDomainCard extends Container {
    * Find displayed number of Participants in this domain.
    */
   async getParticipantsCount(): Promise<string> {
-    const selector = `${this.getXpath()}//*[contains(normalize-space(text()), "participants in domain")]`;
+    const selector = `${this.getXpath()}//*[contains(normalize-space(), "participants in domain")]/div`;
```

Comment:
I suspect removing `text()` was the minimal fix. Fine with the other changes though